website/flight-logs/2023-06-02-cocreate.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Log - Sprint 11 🛫
+description: Flight Log of Co-Creation Activities
+slug: flight-log-11
+tags: [log, sprint ]
+---
+
+## Work In Progress
+- Investigating pushing to a VPC Endpoint. 
```

Comment:
This needs to read as "investigating VPC S3 endpoint notation for the AWSS3Put business process."